### PR TITLE
[React rewrite] - parity with pug version

### PR DESCRIPTION
**Pull request by Niputi**

### CODE REVIEW

**Comment by Niputi** [comment]
"commands" page is last remaining

File: src/pages/about.tsx [hunk 1]
```diff
@@ -0,0 +1,213 @@
+import * as React from "react"
+import { RouteComponentProps } from "@reach/router"
+import NavbarComponent from "../components/navbar"
+import { FooterComponent } from "../components/footer"
+
+const devs = [
```
Comment (velddev) [inline src/pages/about.tsx]: Would be nicer to have these as JSON files

File: src/pages/about.tsx [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+    twitch: "https://www.twitch.tv/velddev",
+  },
+  {
+    name: "Niputi",
+    desc: "Web/backend Developer",
+    uid: "162325985079984129",
+    github: "https://github.com/Niputi",
+  },
+]
+const discordStaff = [
```
Comment (velddev) [inline src/pages/about.tsx]: Same for these, a staff.json would be nicer

File: src/pages/about.tsx [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+    uid: "189158295704043521",
+  },
+  {
+    name: "Homer12346",
+    desc: "Server Maid",
+    uid: "136823052368347136",
+  },
+]
+
+const contributers = [
```
Comment (velddev) [inline src/pages/about.tsx]: Same as above

File: src/pages/donate.tsx [hunk 4]
```diff
@@ -1,13 +1,158 @@
 import * as React from "react"
-import { RouteComponentProps } from "@reach/router";
+import { RouteComponentProps } from "@reach/router"
+import NavbarComponent from "../components/navbar"
+import { FooterComponent } from "../components/footer"
+// import RewardComponent from "../components/cards/reward"
```
Comment (velddev) [inline src/pages/donate.tsx]: Remove the comment if it's not used

Comment (Niputi) [reply src/pages/donate.tsx]: I thought that component was created for the donate page. couldn't find the purpose so commented it out

Comment (velddev) [reply src/pages/donate.tsx]: Ah yeah, I probably was working on that component

Comment (Niputi) [reply src/pages/donate.tsx]: yeah. 6th june

**Comment by velddev** [comment]
Is this PR ready for a replacement of master?

**Comment by Niputi** [comment]
yes if you don't think any changes need to be made.
ps. I have not added anchor links

File: src/data/contributers.ts [hunk 5]
```diff
@@ -0,0 +1,18 @@
+export default [
+  "AlexFlipnote",
+  "Brandon",
+  "Drummss",
+  "Florestina",
+  "Fuzen",
+  "GrammarJew",
+  "Igarashi",
+  "Xetera",
```
Comment (velddev) [inline src/data/contributers.ts]: This array is on alphabetic order

Comment (Niputi) [reply src/data/contributers.ts]: better?

**Pull request merged**